### PR TITLE
Added webpack support

### PR DESCRIPTION
**Pull request by SuperPaintman**
Hi!
I fix exports from module.

```js
const angular = require('angular');
const ngProgressLite = require('ngprogress-lite');

angular.module('App', [
  ngProgressLite // <- exports string with module name
])
```

### CODE REVIEW

**Comment by voronianski** [comment]
@SuperPaintman but it already has UMD support - https://github.com/voronianski/ngprogress-lite/blob/master/ngprogress-lite.js#L11

**Comment by SuperPaintman** [comment]
@voronianski https://github.com/voronianski/ngprogress-lite/blob/master/ngprogress-lite.js#L22

**Comment by voronianski** [comment]
@SuperPaintman but if we just edit this [line](https://github.com/voronianski/ngprogress-lite/blob/master/ngprogress-lite.js#L22) to:

```diff
- angular.module('ngProgressLite', []).provider('ngProgressLite', function () {
+ return angular.module('ngProgressLite', []).provider('ngProgressLite', function () {
```

Will it solve the issue? Or it needs to return string?

**Comment by SuperPaintman** [comment]
I agree, this is also a good way. But we should return the strins.
```

**Comment by voronianski** [comment]
@SuperPaintman then maybe it's better to update your PR accordingly to `return angular.module(..).name` ?

**Comment by SuperPaintman** [comment]
@voronianski yeah, it's right

**Comment by voronianski** [comment]
@SuperPaintman thanks!

**Comment by SuperPaintman** [comment]
@voronianski fixed. Pleas inc `path` version

**Comment by SuperPaintman** [comment]
@voronianski 

`package.json`:
```diff
- "version": "1.0.7",
+ "version": "1.0.8",
```

File: ngprogress-lite.js [hunk 1]
```diff
@@ -6,20 +6,20 @@
...
-    } else if (typeof define === 'function' && define.amd) {
-        // AMD
-        define(['angular'], factory);
-    } else {
-        // Global Variables
-        factory(root.angular);
-    }
+	if (typeof module !== 'undefined' && module.exports) {
+		// CommonJS
+		module.exports = factory(require('angular'));
```
Comment (voronianski) [inline ngprogress-lite.js]: @SuperPaintman what's wrong with your identation?

Comment (SuperPaintman) [inline ngprogress-lite.js]: @voronianski in factory for indentation used `\t`, but in module initial - `    ` (4 space). I fixed this

**Pull request merged**